public/javascripts/SVLabel/src/SVLabel/canvas/Canvas.js

```diff
@@ -333,7 +333,7 @@ function Canvas(ribbon) {
     function setOnlyLabelsOnPanoAsVisible(panoramaId) {
```

Comment:
Oh and this function should probably be renamed then? The name implies that only labels placed on the current pano would be visible :)